### ISSUE TITLE
add config with custom keyboard shortcuts

### ISSUE THREAD

**Issue by arozwalak** [golbin/git-commander#35]
I'm using git through **screen** where `Ctrl-a` is special combination to switch to screen options. So this way I can't use most of the keys from git-commander

**Comment by golbin** [comment]
Actually I faced the problem too. And C-c is also a problem. :-(

So, I've thinking which is better; Adding config or Change default keys.

What do you think?

**Comment by jzsfkzm** [comment]
Adding the ability to configure keys would be great! With custom key mapping you could also provide different basic flavours of key setups, I'm thinking about vi-style paging with C-f/b vs. mc-style paging with PageUp/Down. I'd be happy to work on this, shall I?

**Comment by arozwalak** [comment]
Yes, I think that configurable keys would be better choice so everyone can do it on their way.

**Comment by golbin** [comment]
@jzsfkzm @arozwalak I see. Actually, if we make this configurable, then easy to change the default keys. Thanks!
